js/scripts/generate_imported_types.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
I was expecting this to live some where 🤔 I had sdk would just get a simple `ts` file with all the generated stuff updated and we commit the change.

Reply:
I wanted to leave the language-specific SDKs responsible for generating types in the way they see fit, and the internal repo to only be responsible for producing an openAPI spec.